service/src/main/java/bio/terra/cbas/dependencies/wes/CromwellService.java

```diff
@@ -30,16 +30,19 @@ public class CromwellService implements HealthCheck {
   private final CromwellClient cromwellClient;
   private static final String API_VERSION = "v1";
   private final ApiClient cromwellWriteClient;
+  private final ApiClient cromwellAuthReadClient;
 
   private final CbasNetworkConfiguration cbasNetworkConfiguration;
 
   public CromwellService(
       CromwellClient cromwellClient,
       ApiClient cromwellWriteClient,
-      CbasNetworkConfiguration cbasNetworkConfiguration) {
+      CbasNetworkConfiguration cbasNetworkConfiguration,
+      ApiClient cromwellAuthReadClient) {
```

Comment:
Something to think about and watch in the future - I think this means that every connection to CBAS through (any API handler which ever uses Cromwell) will always start by creating two connections to Cromwell, one auth'd and one not.

I think that's fine, and anyway we're sandboxing CBAS pretty well so the total scale is going to be low..... just something to watch if we ever run into trouble

Reply:
Perhaps we could use the `@Lazy` [annotation](https://www.baeldung.com/spring-lazy-annotation) on the bean if this ever becomes a problem